data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/ReflectionBasedBeaconRestApi.java

```diff
@@ -332,7 +333,7 @@ public ReflectionBasedBeaconRestApi(
   }
 
   @Override
-  public void start() {
+  public SafeFuture<?> start() {
     try {
       app.start();
     } catch (RuntimeException ex) {
```

Comment:
Rather than the `throw`  in the line below here we should change it to be `return SafeFuture.failedFuture(new InvalidConfigurationException(...` so the exception is returned as a failed future rather than thrown.